hugo/content/changelog.html

```diff
@@ -0,0 +1,1260 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><style>body {
```

Comment:
Don't check-in this file, this will get auto generated and pushed to gh-pages branch during the release